encrypted-config-value/src/main/java/com/palantir/config/crypto/algorithm/EncryptedValueSupplier.java

```diff
@@ -40,7 +39,7 @@ EncryptedValue get() throws
         InvalidKeySpecException,
         IOException;
 
-    static EncryptedValue silently(EncryptedValueSupplier supplier) {
+    public static EncryptedValue silently(EncryptedValueSupplier supplier) {
```

Comment:
what would the common util class do? Silently wrap an exception producing supplier?